cql3/statements/create_table_statement.cc

```diff
@@ -465,6 +465,20 @@ std::optional<sstring> check_restricted_table_properties(
                 throw exceptions::syntax_exception(fmt::format("The number of windows ({}) specified is larger than the allowed "
                                                    "number of windows specified by the twcs_max_window_count ({}) parameter.", window_count, max_windows));
             }
+        } else {
+              switch (qp.db().get_config().restrict_twcs_default_ttl()) {
```

Comment:
I think the name "restrict_twcs_default_ttl" is a bit confusing, because turning it to true doesn't restrict using default_ttl - it requires default_ttl. It restricts the ability *not* to use it. So I wonder if the name "restrict_twcs_without_default_ttl" would be more accurate?